.github/ISSUE_TEMPLATE/new_resource.yml

```diff
@@ -99,3 +62,39 @@ body:
         - "12"
     validations:
       required: true
+  - type: input
+    id: title
+    attributes:
+      label: Title
+      description: The title is used in list views on the home page and [https://calitp.org/resources](https://calitp.org/resources)
+    validations:
+      required: true
+  - type: input
+    id: link
+    attributes:
+      label: Asset URL
+      description: The URL of the resource
+      placeholder: ex. link to a PDF from Google Drive
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Category
+      description: What category is this resource in?
+      multiple: false
+      options:
+        - Case studies
+        - Fact sheets & overviews
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Tag
+      description: What tag is this resource in?
+      multiple: false
```

Comment:
We can support multiple here right?

Reply:
<img width="471" alt="image" src="https://github.com/cal-itp/calitp.org/assets/3673236/48499a93-2ef1-4e3f-b3cc-5161ee29a38b">